src/translations/en.json

```diff
@@ -2033,26 +2040,33 @@
               "geo_location_sources": "Geolocation Sources",
               "dark_mode": "Dark Mode?",
               "default_zoom": "Default Zoom",
-              "source": "Source"
+              "source": "Source",
+              "description": "A card that allows you to display entities on a map."
```

Comment:
`A` or `The`?

Furthermore, this description doesn't mention the name of the card in the description, the others do.